src/utils/orgUtils.ts

```diff
@@ -18,6 +18,13 @@ export interface Field {
     label: string;
     type: string;
 }
+
+export interface CompactLayoutField {
+    editableForNew: boolean;
+    editableForUpdate: boolean;
+    label: string;
+}
```

Comment:
@dbreese We don't have to use this interface. I added so that I can cast to this before returning to the caller so LWC generating code can have defined fields in the elements of returned array of sObject's fields. Otherwise, you might need to do lot of casting as I do in `getCompactLayoutFieldsForSObject`.